### ISSUE TITLE
[HelpHub] Audio block

### ISSUE THREAD

**Issue by femkreations** [WordPress/Documentation-Issue-Tracker#339]
Article:  https://wordpress.org/documentation/article/audio-block/

## Updating for 6.2
- [ ] Add toolbar button to add/remove caption. [45112](https://github.com/WordPress/gutenberg/pull/45112)

## Updating for 6.1
- [ ] Add: content role to relevant audio block attributes. [44037](https://github.com/WordPress/gutenberg/pull/44037)
- [ ]  Display upload error notices using snackbars. [43891](https://github.com/WordPress/gutenberg/pull/43891)
- [ ]  Add margin and padding supports to Audio block. [43351](https://github.com/WordPress/gutenberg/pull/43351)
- [ ] Remove redundant block name from "settings" panels [40275](https://github.com/WordPress/gutenberg/pull/40275)

## Updating for 6.0
- [ ] Audio: Add uploading state. [37739](https://github.com/WordPress/gutenberg/pull/37739)

## General
- [ ] Make sure all screenshots are relevant to the latest version 
- [ ] Make sure videos, if any, are up to date 
- [ ] Add ALT tags for the images
- [ ] Make sure the headings are in sentence case
- [ ] Convert all reusable blocks to a ‘regular block’.
- [ ] Update the changelog at the end of the article

## Notes
1. We are not adding detailed updates for the Block settings - Typography, Color, Layout, Dimension and Border and for the More Options setting in the block toolbar. Pls mention the settings included for the block and link to the detailed settings pages instead.
2. Pls make 6.2 updates first and then move to 6.1.
3. Some of the PRs have great videos that can be added to the article
4. For 6.1, you can pick the screenshots added in the comments previously, if they are relevant.

**Comment by pls78** [comment]
I'm working on this! :)

**Comment by pls78** [comment]
Here is the updated video:
https://user-images.githubusercontent.com/68744851/168261110-0cd9c47c-cd95-4830-88ac-d4c8d4cdbaf1.mp4

Screenshots follow, the number in the filenames (and the order in which they appear here) is related to the order of the screenshots in the doc.
<img width="524" alt="Screenshot 1" src="https://user-images.githubusercontent.com/68744851/168261377-14b81491-90c9-4ec3-92c6-fc1c11df2e54.png">
<img width="223" alt="Screenshot 2" src="https://user-images.githubusercontent.com/68744851/168261398-6fb0a6c8-b6dd-4c85-bcda-e105fc6248f8.png">
<img width="644" alt="Screenshot 3" src="https://user-images.githubusercontent.com/68744851/168261419-2b5655c8-eeeb-4604-81dd-be5925ce0dd8.png">
<img width="647" alt="Screenshot 4" src="https://user-images.githubusercontent.com/68744851/168261431-e2112499-5edd-4768-99e8-cf49c250a103.png">
<img width="642" alt="Screenshot 5" src="https://user-images.githubusercontent.com/68744851/168261449-2c2591b3-56c1-4a4a-a3b6-fafe78521a10.png">
<img width="279" alt="Screenshot 6" src="https://user-images.githubusercontent.com/68744851/168261461-a5f6228c-eef3-4966-927c-df57a1a4cef4.png">
<img width="183" alt="Screenshot 7" src="https://user-images.githubusercontent.com/68744851/168261472-3b3ebaa2-d68e-4d46-847f-5b10f1bc0956.png">
<img width="284" alt="Screenshot 8" src="https://user-images.githubusercontent.com/68744851/168261490-979325ab-a769-4b34-b8df-6caf5300ad83.png">
<img width="285" alt="Screenshot 9" src="https://user-images.githubusercontent.com/68744851/168261502-dee652f0-10ef-427f-858d-f911b4e3b0e5.png">

**Comment by zzap** [comment]
Thank you @pls78 :heart:

**Comment by abigail0r** [comment]
I'm working on this

**Comment by lnaima** [comment]
I'll work on this

**Comment by lnaima** [comment]
Only one menu is slightly different (an icon), here's the screenshot. The reusable block icon is changed.
![Audio-block_More_options](https://user-images.githubusercontent.com/6752710/190122925-2d2c69a0-604a-471d-a1f1-56369347ae66.png)

Like @abigail0r I think there are some headings missing.

**Comment by femkreations** [comment]
Added tasks for 6.1

## Notes
1. We are not adding detailed updates for the Block settings as we are revamping the block settings for all the block articles 
2. Pls make 6.1 updates first and then move into 6.0.
3. Some of the PRs for 6.1 have great videos that can be added to the article (even for the Block settings section)
4. For 6.0, you can pick the screenshots added in the comments previously.

**Comment by carolinan** [comment]
I will start working on this issue today.

**Comment by carolinan** [comment]
I have scheduled the update for April 27, 2023 9:11 am UTC+0, hoping that will give enough time to review it. If not, I can login and change the time.

- Updated screenshots and video
- Added alt texts
- Fixed a minor typo
- Added a new section about the updated caption options
- Added a new section called troubleshooting:
I added this section because "Display upload error notices using snackbars." is in the todo list in the issue description.
The section includes short information and a new screenshot of the error notice.

I did not add information about padding and margin because it was already included.
